backend/seed_data/hints.yaml

```
@@ -74,3 +74,24 @@ hints:
+      - text: "Once you find a land cell, perform DFS on its neighbors to mark all connected land cells as visited."
+        order: 3
+
+  - module_type: "BONUS_CHALLENGE"
+    module_title: "Max Area of Island"
+    hints:
+      - text: "Use depth-first search (DFS) to traverse the grid."
+        order: 1
+      - text: "Track the size of the current island as you explore it."
+        order: 2
+      - text: "Perform DFS on land cell's neighbors and sum the areas."
+        order: 3
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)